app/client/cli/cache/session.go

```diff
@@ -0,0 +1,73 @@
+package cache
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/pokt-network/pocket/persistence/kvstore"
+	"github.com/pokt-network/pocket/rpc"
+)
+
+var errSessionNotFound = errors.New("session not found in cache")
+
+// SessionCache stores and retrieves sessions for application+relaychain pairs
+//
+//	It uses a key-value store as backing storage
+type SessionCache struct {
```

Comment:
Might be worth adding a method that calls `store.Stop()` or doing `defer store.Stop()` in the existing methods so the cache doesn't remain open indefinietly

Reply:
Thank you for the review. Fixed.